app/controllers/projects_controller.rb

```diff
@@ -195,6 +199,19 @@ def request_join
     end
   end
 
+  def submit_fairdata_station
+    path = params[:datastation_data].path
+    policy = Policy.new
+    policy.set_attributes_with_sharing(policy_params)
+    datadata_inv = Seek::FairDataStation::Reader.instance.parse_graph(path)
+    @investigation = Seek::FairDataStation::Writer.instance.construct_isa(datadata_inv.first, current_person, [@project], policy)
+    @investigation.save!
+
+    respond_to do |format|
+      format.html { redirect_to(@investigation) }
+    end
+  end
+
```

Comment:
if the TTL file contains syntax errors, how to handle it?

Reply:
currently it will throw an error, but that is fine for now. The TTL is being generated by FDS so unlikely to happen